eelbrain/_stats/testnd.py

```diff
@@ -1853,12 +1855,18 @@ def __init__(self, y, match=None, sub=None, ds=None,
         v_dim = ct.y.dimnames[cdist._vector_ax + 1]
         v_mean = ct.y.mean('case')
         v_mean_norm = v_mean.norm(v_dim)
-        cdist.add_original(v_mean_norm.x if v_mean.ndim > 1 else v_mean_norm)
+        if use_t2_stat:
+            t2_map = self._vector_t2_map(ct.y)
+            cdist.add_original(t2_map.x if v_mean.ndim > 1 else v_mean_norm)
```

Comment:
```suggestion
            cdist.add_original(t2_map.x if v_mean.ndim > 1 else t2_map)
```

This is because if you run the test with a variable where the only dimension other than case is space, then the output of the test function should be a scalar

Reply:
By the way, it took me a while to figure out there exists np scalar arrays, and how to declare one. :(

Reply:
Yeah they caused me some headaches too when I first encountered them :)